pkg/packagekit/updates.jsx

```diff
@@ -470,11 +425,8 @@ class ApplyUpdates extends React.Component {
                         <div className="spinner spinner-xs spinner-inline" />
                         {actionHTML}
                     </div>
-                    <div className="progress progress-label-top-right">
-                        <div className="progress-bar" role="progressbar" style={ { width: this.state.percentage + "%" } }>
-                            { this.state.timeRemaining !== null ? <span>{moment.duration(this.state.timeRemaining * 1000).humanize()}</span> : null }
-                        </div>
-                    </div>
+                    <Progress title={this.state.timeRemaining && moment.duration(this.state.timeRemaining * 1000).humanize()} value={this.state.percentage} />
```

Comment:
This part would fit better into the first commit, it is unrelated to the header bar changes. (But not that important).